### ISSUE TITLE
Consider moving resources.eahub.org to eahub.org with chapters.1fortheworld.org

### ISSUE THREAD

**Issue by viktor-yunenko** [rtcharity/eahub.org#1240]
Martyna would like to join the resources navbar and our navbar, to avoid losing users when they go to resources.eahub.org, because there's no way back.

At the same time I was thinking about adding a CMS component to the hub, which would allow us to edit pages as about-us from the admin interface, without the need of deployments and PR reviews. That could be a good time to quickly do an MVP of that kind, I'm happy to do the migration myself if needed.

**Comment by sebgrebe** [comment]
AFAIA the idea was to move Resources into the EA Forum? @vagarwalla

**Comment by vagarwalla** [comment]
Possibly, they don't have concrete plans. I don't think we should invest time in this until we've finished our current tasks. And had a word with Catherine.  

There are definitely a lot of improvements to be made,  but we need to prioritise !

**Comment by viktor-yunenko** [comment]
I agree, I created the ticket only because it can simplify our life somewhere down the road, when we do the navbar redesign.

For the context: Martyna would like to join the resources navbar and our navbar, to avoid losing users when they go to resources.eahub.org, because there's no way back.

At the same time I was thinking about adding a CMS component to the hub, which would allow us to edit pages as about-us from the admin interface, without the need of deployments and PR reviews.
